source/reference/api/start.txt

```diff
@@ -137,25 +137,20 @@ Request Body Parameters
    * - ``includeNamespaces``
      - array
      - Optional
-     - Filters the databases or collections to sync. The filter syntax
-       is:
-
-       .. include:: /includes/api/facts/includeNamespaces-syntax.rst
+     - Filters the databases or collections to include in sync. 
 
        .. versionadded:: 1.1
```

Comment:
[suggestion] I'd recommend moving this to the bottom of the description to maintain consistency with all the other entries in the table. Same goes for line 151.

Reply:
Fixed